### ISSUE TITLE
Strange values after loading FCS3.1 file from Fortessa x20

### ISSUE THREAD

**Issue by chofski** [taborlab/FlowCal#321]
I am attempting to load a file generated on a Fortessa x20 analyser which is in FCS3.1 format and am finding that the raw values loaded are very strange. Lots of negative values and converting to a.u. doesn't seem to help. I suspect it might be an issue with the file format and pulling out the wrong data type, but was wondering if this is a known issue and whether it can be easily fixed. Same problem does not occur with the fcs_parser module.

**Comment by castillohair** [comment]
Hi Thomas,

This is the first time I'm hearing of an issue like you're describing. Could you send an example fcs file so we can investigate further? 

Thanks.

**Comment by JS3xton** [comment]
Hey Tom,

I think the behavior you've described is the correct behavior, and it appears to me that `FlowCal` and `fcsparser` behave the same in this regard.

I have a Fortessa x20 FCS3.1 file from an old project, and both `FlowCal` and `fcsparser` yield the same number of negative values across all channels. I noticed, however, that only "area" channels (whose names end with "-A") have negative values. I believe modern machines do a baseline correction / background subtraction in the electronics prior to data storage, and this gives rise to these negative values. I've heard complaints of this around our lab, but I don't think we've tried in earnest to address it.

Also, converting to a.u. doesn't change the data from my file, and I think this is again correct behavior. The data type for my file is float (`$DATATYPE='F'`), so conversation to a. u. (`FlowCal.transform.to_rfi()`) simply divides by the amplifier gain (usually stored in the `$PnG` parameter), which is 1.0 for almost all of the channels in my file.

Hope this helps!
